GoogleForCreators/web-stories-wp
# Right Click Menu: Add tracking

## Context

Right Click Menu

## Summary

Add tracking to the right click menu.

## Relevant Technical Choices

Did not add tracking to functions that are being removed in #8679.

## To-do

n/a

## User-facing changes

n/a

## Testing Instructions

Verify that tracking events are sent on:
- menu open
- action click
- snackbar action click

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

Yes!
- Tracks when the menu is opened
- Tracks when a button in the menu is clicked
- Tracks when a user clicks the `Undo` button in snackbars that are triggered from the right click menu actions

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #8681
